lib/portlayer/storage/nfs/volume.go

```diff
@@ -107,6 +107,7 @@ func (v *VolumeStore) VolumeCreate(op trace.Operation, ID string, store *url.URL
 	}
 
 	u, _ := v.Service.URL()
+	u.Scheme = "nfs"
```

Comment:
IIRC this is assigned [here](https://github.com/vmware/vic/blob/master/lib/portlayer/storage/nfs/target.go#L73).  Just check the `Scheme` there and throw an error if it's not `nfs`.  Then you can drop this line here.